docker-storage-setup.sh

```diff
@@ -410,6 +412,18 @@ check_docker_storage_metadata() {
   exit 1
 }
 
+reset_docker_root_volume () {
+  if docker_root_volume_exists; then
+     if ! get_docker_root_dir; then
+        return 1
+     fi
```

Comment:
We should not have to look at docker root dir again. We can just check if root volume is mounted and where it is mounted. Just unmount it and delete volume.